### DISCUSSION TITLE
Can MacOS provide an installation method for Homebrew?

### DISCUSSION THREAD

**Discussion by wiwiluo** [Martchus/syncthingtray#242] [General]
MacOS currently only provides the installation method of macports, but there may be many problems when using macports and homebrew at the same time. Can you provide an additional installation method of homebrew?

**Comment by Martchus** [comment]
> Can you provide an additional installation method of homebrew?

I personally won't invest any effort into that (similar to how I cannot take the effort for providing packages for any GNU/Linux distribution and Windows package manager out there).

If someone is willing to maintain a Homebrew package (like people already maintain packages for certain GNU/Linux distributions and Windows package managers) I'll of course help resolving any potential (build) problems. I already try to avoid introducing code the would break portability so probably there wouldn't be much to fix. (Please include the whole CMake invocation and full error logs in case something goes wrong when filing an issue.)

Note that the version on MacPorts is probably *very* outdated. At this point I recommend to build from sources on Mac.
